mage_integrations/mage_integrations/tests/sources/bigquery/test_bigquery.py

```diff
@@ -0,0 +1,137 @@
+from google.cloud.bigquery import Row
+from mage_integrations.sources.bigquery import BigQuery
+from unittest.mock import MagicMock, patch
+import os
+import unittest
+
+
+def build_sample_bigquery_rows():
+    return [
+        Row(
+            ('demo_users', 'NULL', None, 'id', 'STRING(100)', 'NO'),
```

Comment:
In unit test, let's try creating more variants in the test data, like more column types in the sample rows.